starlette/applications.py

```diff
@@ -78,6 +79,7 @@ def __init__(
         )
         self.user_middleware = [] if middleware is None else list(middleware)
         self.middleware_stack: typing.Optional[ASGIApp] = None
+        self.request_max_size = request_max_size
```

Comment:
It might be a good idea to start making these attributes private.

Reply:
Not true: https://github.com/encode/starlette/blob/2168e47052239da5df35d5353bb986f760c51cef/starlette/datastructures.py#L61

Reply:
I am referring to the `Starlette` class itself
https://github.com/encode/starlette/blob/master/starlette/applications.py#L71